src/daemon.js

```diff
@@ -78,30 +78,38 @@ function parseConfig (path, callback) {
   ], callback)
 }
 
+function tempDir (isJs) {
+  return join(os.tmpdir(), `${isJs ? 'jsipfs' : 'ipfs'}_${String(Math.random()).substr(2)}`)
+}
+
 /**
  * Controll a go-ipfs node.
  */
 class Node {
```

Comment:
Rename the class to `GoDaemon`

Reply:
I forgot to change the comment, this now controls both, go and js nodes.